### PR TITLE
Indie Game: Life After added

### PR DESCRIPTION
**Pull request by wseemann**
I realized they made a sequel to Indie Game: The Movie so I'm adding that too.

### CODE REVIEW

File: documentary_prebuilt.md [hunk 1]
```diff
@@ -432,6 +432,16 @@
 * Country: USA
 * [_IMDb_](https://www.imdb.com/title/tt1942884/)
 
+#### Indie Game: Life After
```
Comment (greybax) [inline documentary_prebuilt.md:435]: please add title under contents section

**Review by greybax — CHANGES_REQUESTED** [review]

File: documentary_prebuilt.md (hunk 1 above)
Comment (wseemann) [reply documentary_prebuilt.md:435]: I'm not sure what you mean, can you elaborate? This PR seems to follow the template listed in the contribution guidelines:

#### Matrix 
:star: Matrix_Score

> A computer hacker learns from mysterious rebels about the true nature of his reality and his role in the war against its controllers.

Matrix
* Year: 1999
* Country: USA, Australia
* [_IMDb_](https://www.imdb.com/title/tt0133093/)

   - The first time I watched this, I was absolutely amazed with the concept of it. The action is just so amazing to watch. I've seen this film couple times and each time it just gets better. If you like sci fi action films then this is the film for you.
([@greybax](https://github.com/greybax))

Comment (wseemann) [reply documentary_prebuilt.md:435]: Can this be merged or does it require additional changes?

Comment (greybax) [reply documentary_prebuilt.md:435]: Please add movie title into contents section:
```
- [6.0-6.5 :star:](#_60-65)
.......
- [5.9-5.5 :star:](#_59-55)
  - [Indie Game: Life After](#indie-game-life-after)
```

**Review by greybax — APPROVED** [review]

**Comment by greybax** [comment]
Thank you for you PR @wseemann! Merged

**Pull request merged**